pkg/registry/core/shoot/strategy.go

```diff
@@ -168,32 +166,7 @@ func (shootStrategy) Validate(_ context.Context, obj runtime.Object) field.Error
 	return allErrs
 }
 
-func (shootStrategy) Canonicalize(obj runtime.Object) {
-	shoot := obj.(*core.Shoot)
-
-	cleanupAdmissionPlugins(shoot)
-}
-
-func cleanupAdmissionPlugins(shoot *core.Shoot) {
-	var (
-		admissionPlugins      []core.AdmissionPlugin
-		shootAdmissionPlugins = shoot.Spec.Kubernetes.KubeAPIServer.AdmissionPlugins
-	)
-
-	kubernetesVersion, err := semver.NewVersion(shoot.Spec.Kubernetes.Version)
-	if err != nil {
-		return
-	}
-
-	for _, plugin := range shootAdmissionPlugins {
-		if constraint, ok := admissionpluginsvalidation.PluginsInMigration[plugin.Name]; ok && constraint.Check(kubernetesVersion) {
-			continue
-		}
-
-		admissionPlugins = append(admissionPlugins, plugin)
-	}
-
-	shoot.Spec.Kubernetes.KubeAPIServer.AdmissionPlugins = admissionPlugins
+func (shootStrategy) Canonicalize(_ runtime.Object) {
```

Comment:
Nit
```suggestion
func (shootStrategy) Canonicalize(_ runtime.Object) {}
```

Reply:
It is this way for all the other object strategy's